docs/mpdev-reference.md

```diff
@@ -0,0 +1,57 @@
+## Setup
+
+TODO: Give instructions for installing mpdev.
+
+## Mpdev resources
+
+The `mpdev` tool creates resources from yaml templates. The mpdev resource schema
+is inspired by the schema for kubernetes resources, where a resource type is
+uniquely specified by a `kind` and `apiVersion`.
+
+Currently, the `mpdev` tool supports two types of resources,
```

Comment:
Change to `the following types of resources` and turn this into a list?

Reply:
Fixed below